src/libraries/System.Security.Principal.Windows/ref/System.Security.Principal.Windows.cs

```diff
@@ -210,7 +208,6 @@ public enum WellKnownSidType
         WinCapabilityEnterpriseAuthenticationSid = 93,
         WinCapabilityRemovableStorageSid = 94,
     }
-    [System.Runtime.InteropServices.ComVisibleAttribute(true)]
```

Comment:
Have we removed ComVisibleAttributes from elsewhere?  I can't remember now.

Reply:
Yes, we have deleted it from (close to) everywhere. This one looks like oversight. It should be fine to delete it both from here and from the implementation.

Reply:
Done